documentation/Megaform Sequence Diagram.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this? Does this need to be under revision control?

Reply:
I see now that this is the sequence diagram drawing. Can we make it a file type other than svg? This file type is a strange one to put under revision control (plus, it's basically one super long line so we should at least format it to be readable)

Reply:
I was planning to put the image in the document template and upload it as pdf if that's fine.

Reply:
Yeah, that's what I'd expected, too